n2y/main.py

```diff
@@ -24,6 +24,13 @@ def main(raw_args, access_token):
         formatter_class=argparse.RawTextHelpFormatter,
     )
     parser.add_argument("config", help="The path to the config file")
+    parser.add_argument(
+        '--max_retries', '-m', default=5,
```

Comment:
We have the default set to 5 in multiple places. Let's extract the 5 to a constant somewhere and use that everywhere. That way, we can just change it in one place when we need to.